### DISCUSSION TITLE
使用跳板服务器时一直报错无法连接

### DISCUSSION THREAD

**Discussion by dlzhongp** [lollipopkit/flutter_server_box#481] [General]
认证失败，请检查密码/密钥/主机/用户等是否错误
SSHAuthAbortError(Connection closed before authentication)

**Comment by hail100** [comment]
@lollipopkit 遇到同样问题。

**Comment by exstntlmsnthrp** [comment]
I also have this problem on the newest version 1104. 

我在最新版本1104上也遇到了这个问题。
